act-rules-format.bs

```diff
@@ -140,43 +140,37 @@ Component properties starting with `aria-` MUST exist in the list of WAI-ARIA at
 ACT Test Procedure {#test-proc}
 ===============================
 
-Selector {#test-proc-selector}
-------------------------------
+Applicability
+-------------
 
-A selector is a filter procedure that is used on the input data to be evaluated against the test procedure. For example, finding specific nodes in the DOM tree, or finding tags that are incorrectly closed in an HTML document.
+The applicability section is a required part of an ACT rule. It MUST contain a precise description of what (parts of the) test subject the rule applies to. For example, specific nodes in the DOM tree, or tags that are incorrectly closed in an HTML document. These are known as the *test targets*.
 
-Selector syntax depends on the test subject type. When a formal syntax can be used, that (part of the) selector should use that syntax. When a formal selector syntax can not be used, that (part of the) selector MAY be described unambiguously in plain language.
+Applicability MUST be described objectively, unambiguously and in plain language. When a formal syntax (such as a CSS selector or XPATH) can be used, that (part of the) applicability MAY use that syntax in addition to the plain language description. While testing, if nothing on in the test subject matches the applicability of the rule, the result is `inapplicable`.
```

Comment:
"if nothing on in"; should the "on" be removed and "in" changed to "within"?

Reply:
Agreed

Reply:
agreed

Reply:
Editorial: Grammatical error in the first sentence. I suggest, "The applicability section is a required part of an ACT rule. It MUST contain a precise description of the parts of the test subject to which the rule applies."